openwisp_controller/config/migrations/0035_extend_openwisp_users.py

```diff
@@ -0,0 +1,52 @@
+import django.db.models.deletion
+from django.db import migrations, models
+from swapper import get_model_name
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [('config', '0034_template_required')]
+
+    operations = [
+        migrations.AlterField(
+            model_name='organizationconfigsettings',
+            name='organization',
+            field=models.OneToOneField(
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name='config_settings',
+                to=get_model_name('openwisp_users', 'Organization'),
+                verbose_name='organization',
```

Comment:
let's not generate a new migration just for code fixes to the migration, please go back into the previous migrations and change the code of the first migration in which each field appears.

Reply:
Sure